src/main/display.cc

```diff
@@ -1184,28 +1181,7 @@ GTKChapDisp::getVerseBefore(SWModule &imodule)
 		key->setChapter(i * chapter);
 		key->setVerse(0);
 
-#if 0 // with footnote counting, we no longer cache before/after verses.
-		ModuleCache::CacheVerse& cVerse = ModuleMap
-		    [ModuleName]
-		    [key->Testament()]
-		    [key->Book()]
-		    [key->Chapter()]
-		    [key->Verse()];
-	
-		if (!cVerse.CacheIsValid(cache_flags))
-			cVerse.SetText((strongs_or_morph
-					? block_render(imodule.renderText().c_str())
-					: imodule.renderText().c_str()),
-				       cache_flags);
-		buf = g_strdup_printf("%s<br />", cVerse.GetText());
-#else
-# if 0
-		buf = g_strdup_printf("%s<br />",
-				      (strongs_or_morph
-					   ? block_render(imodule.renderText().c_str())
-					   : imodule.renderText().c_str()));
-# else
-		// gosh this is gross.
+		// begin gosh this is gross.
```

Comment:
Why you do this unrelated changes?

Reply:
It isn't unrelated. Without the wrapping `#if 0 ... else ... end` block that got removed, it was no longer clear what section of code the remaining comment referred to. I defined the region to not loose the little bit of information provided by the fact that this code was previously grouped together.